README.md

```diff
@@ -186,7 +186,7 @@ More info on [Portainer](https://www.portainer.io/).
     "motd": "Server is currently offline"
   },
   "callbackServer": {
-    "url": "https://mc.example.com/callback",
+    "url": "https://mc.example.com/webhook",
```

Comment:
This isnt the only callback which is the README file, dont we also wanna change the others?

Reply:
Good point, but I think the README need a general overhaul. Especially the webhook part.

Reply:
Yea thats right, lets do that at the end of the refactor